lightly/active_learning/agents/agent.py

```diff
@@ -157,7 +157,7 @@ def added_set(self):
         # added set is new preselected set minus the old one
         added_tag_bitmask = self._preselected_tag_bitmask - self._old_preselected_tag_bitmask
         return added_tag_bitmask.masked_select_from_list(
-            self.api_workflow_client.filenames_on_server
+            self.api_workflow_client.download_filenames_from_server()
```

Comment:
would call it `get_filenames_from_server` and not `download_filenames...`

Reply:
or even `get_filenames` which would be consistent with the names of the other functions, e.g. `get_datasets`